### PR TITLE
Release 1.24.7

### PR DESCRIPTION
**Pull request by amotl**
## About

For releasing a new version based on the recent updates.

/cc @seut

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -6,6 +6,14 @@ Unreleased
 ==========
 
 
+2024-01-12 1.24.7
+=================
+
+- Fixed grammar in ES version of ``DISCOURSE_MSG``. Thanks, @hlcianfagna.
```
Comment (matriv) [inline CHANGES.rst:12]: is `DISCOURSE_MSG` a user facing variable? if not please rephrase, using something that a plain user understands.

Comment (hlcianfagna) [reply CHANGES.rst]: I amended this now, feel free to unresolve if you think it needs further changing

**Review by matriv — APPROVED** [review]
thx! left a minor comment.

File: CHANGES.rst [hunk 2]
```diff
@@ -6,6 +6,14 @@ Unreleased
 ==========
 
 
+2024-01-12 1.24.7
+=================
+
+- Fixed minor grammar issue on the Spanish version of the Help page. Thanks, @hlcianfagna.
```
[the remark this answers is not in the document]
Comment (hlcianfagna) [reply CHANGES.rst]: Good point, I fixed it now

[the remark this answers is not in the document]
Comment (matriv) [reply CHANGES.rst]: thx a lot!

**Comment by amotl** [comment]
Thanks for the review. I carried over your suggestions over to GH-835, which replaces this PR, to run the release without all the other updates, as requested. Therefore, I am closing this one.

**Pull request closed without merging**